### ISSUE TITLE
Running with Rails

### ISSUE THREAD

**Issue by kakoni** [chaps-io/gush-control#18]
Trying to run this under rails. 
in routes I've got 

```
mount Gush::Control::App, at: '/gush'
```

But asset loading fails, 

```
ActionController::RoutingError (No route matches [GET] "/js/application.js"):
```

Which is bit weird since shouldnt sinatra-assetpack work with subpaths (https://github.com/rstacruz/sinatra-assetpack/pull/104)

**Comment by pokonski** [comment]
Good catch, I haven't mounted the Gush Control inside rails so that is something to improve
